scipy/meson.build

```diff
@@ -379,7 +378,6 @@ py3.install_sources(
 #_cython_tree = declare_dependency(sources: [
 _cython_tree = [
   fs.copyfile('__init__.py'),
-  fs.copyfile('linalg.pxd'),
   fs.copyfile('optimize.pxd'),
   fs.copyfile('special.pxd'),
```

Comment:
I suppose we can clean up optimize and special as well?

Reply:
Yes please, treating them all the same is better.